src/main/java/com/es/phoneshop/dao/CustomProductDao.java

```diff
@@ -82,6 +82,41 @@ public List<Product> findProducts() {
         }
     }
 
+    @Override
+    public List<Product> findProductsByDescription(String description) {
+        if (description == null || description.isBlank()) {
```

Comment:
StringUtils.isEmpty

Reply:
would it be better if i used StringUtils.isBlank? (to not allow strings, containing only whitespaces)

Reply:
actually I would not even allow to search with whitespaces, but some websites allow searching with whitespaces (and they return just plp), some sites return smth like "Nothing was found", so the final desicion is up  to you
the comment was more about util itself - while working with String-s it's very convenient to keep in mind this StringUtils class not to create these checks for null and empty strings